client/tests/integration/roles.rs

```diff
@@ -4,7 +4,7 @@ use std::{str::FromStr as _, time::Duration};
 
 use eyre::{eyre, Result};
 use iroha_client::client::{self, Client};
-use iroha_core::{prelude::AllowAll, smartcontracts::permissions::ValidatorBuilder};
+use iroha_core::{prelude::AllowAll, smartcontracts::permissions::builder::ValidatorBuilder};
```

Comment:
I think we shouldn't repeat the name-space in the name. So either 
`permissions::builder::Validator` or `permissions::ValidatorBuilder` is preferred.